src/components/styles/Profile.css

```diff
@@ -1,14 +1,38 @@
-.profile {
+.profile-container {
+  display: flex;
+  justify-content: space-between;
+  padding: 0 8rem;
+}
```

Comment:
Kindly fix the design:
->The profile-container height should be 100%.
Please check the image below->

<img width="912" alt="ss1" src="https://github.com/Zafron047/space_travelers_hub/assets/88501552/6af7b55d-967e-4429-ad21-30c1f70a30b3">

Reply:
Resolved with min-height: 100vh.